src/esp/gfx/Viewer.h

```diff
@@ -11,11 +11,15 @@
 #include <Magnum/Platform/GlfwApplication.h>
 #endif
 #include <Magnum/SceneGraph/Camera.h>
+#include <Magnum/Timeline.h>
 
 #include "esp/agent/Agent.h"
 #include "esp/assets/ResourceManager.h"
 #include "esp/gfx/RenderCamera.h"
 #include "esp/nav/PathFinder.h"
+#include "esp/physics/PhysicsManager.h"
+#include "esp/physics/RigidObject.h"
+//#include "esp/physics/bullet/BulletPhysicsManager.h"
```

Comment:
Should this be deleted or does it need an `#ifdef <thing>` guard?

Reply:
Only the Bullet specific classes need ifdef right now. We didn't need that include here, so removed.